<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>554</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionTransfers"/>
   <addaction name="actionSearch"/>
   <addaction name="actionOptions"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/transfers/add.png</normaloff>:/Icons/transfers/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add torrent file...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Preferences.png</normaloff>:/icons/Preferences.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/ConnectDoBig.png</normaloff>:/icons/ConnectDoBig.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionTransfers">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Transfer.png</normaloff>:/icons/Transfer.png</iconset>
   </property>
   <property name="text">
    <string>Transfers</string>
   </property>
   <property name="toolTip">
    <string>Transfers</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Search.png</normaloff>:/icons/Search.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Search</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
